src/components/layout/Layout/Layout.tsx

```diff
@@ -0,0 +1,24 @@
+import { Outlet } from 'react-router-dom';
+import NavBar from '../Navbar/Navbar';
+import Footer from '../Footer/Footer';
+import './styles.css';
+
+function Layout() {
+  return (
+    <>
+      <div className="header">
+        <NavBar />
+      </div>
+      <div className="body">
+        <div className="container">
+            <Outlet />
+        </div>
+        <div className="footer">
+          <Footer />
+        </div>
+      </div>
+    </>
+  );
+}
```

Comment:
this allows us to add headers/footers/side panels to every page. children components (pages) get happened in `Outlet`